get.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require "connect.php";
+
+function get($db) {
+
+$query = $db->prepare("SELECT `picture`, `species`, `color`, `frequency`, `shit` FROM `birds`;");
+
+$query->setFetchMode(PDO::FETCH_ASSOC);
+
+$query->execute();
+
+$result = $query->fetchAll();
+
+return $result;
+
+}
+
+$result = get($db);
```

Comment:
Remove this and instead call the get() function in your index.php just before you call display()

Reply:
so this? display(get($db))

this stuff still confuses me so thought easier for my brain to put it in a variable. did you mean something else?